test/clj/quil/snippet.clj

```diff
@@ -14,9 +16,42 @@
 (def tests-in-set 50)
 (def current-test (atom -1))
 
+(defn skip-automated-compare? [snippet-name]
+  (contains? #{"current-frame-rate-target-frame-rate"
+               "noise"
+               "noise-detail"
+               "random-2d"
+               "random-3d"
+               "random"
+               "random-gaussian"
+               "resize-sketch"
+               "save"
+               "time-and-date"} snippet-name))
+
+(defn- compare-images [expected actual difference]
+  (let [{:keys [err]} (sh/sh "compare" "-metric" "mae" expected actual difference)
+        result        (second (re-find #"\((.*)\)" err))]
```

Comment:
What structure "err" has that you can extract and parse clojure from it?

Reply:
`err` is a string with something like this: `23.45 (0.115)`. The second number is what I'm extracting.

Reply:
done